src/components/header/navMenu.js

```diff
@@ -11,6 +11,10 @@ const NavMenu = ({
   selectedTeam: { teamColor, urlName },
   playerId,
 }) => {
+  const activeStyle = {
+    pointerEvents: 'none',
+    color: teamColor,
+  };
```

Comment:
I believe this was meant for PR #130, whoops again.
Try to keep your PRs clean when switching between them.

Reply:
nvm, I see what you did, consider resolved